dotcom-rendering/src/web/components/MiniCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/react';
+import { body, breakpoints } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { DCRContainerPalette } from 'src/types/front';
+import type { TrailType } from 'src/types/trails';
+import { ContainerOverrides } from '../../types/palette';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+import { generateSources } from './Picture';
+
+const linkStyles = css`
+	${body.medium()}
+	font-weight: bold;
+	display: flex;
+	align-items: flex-start;
+`;
+
+const linkOverrideStyles = (containerOverrides?: ContainerOverrides) => css`
+	${containerOverrides && `color: ${containerOverrides.text.cardHeadline}`}
+`;
+
+type Props = {
+	trail: TrailType;
+	showImage: boolean;
+	containerPalette?: DCRContainerPalette;
+};
+
+type MiniCardPictureProps = {
+	image: string;
+	alt: string;
+};
+
+const MiniCardPicture = ({ image, alt }: MiniCardPictureProps) => {
+	const sources = generateSources(image, [
+		{ breakpoint: breakpoints.desktop, width: 120 },
+	]);
+
+	const [source] = sources.slice(-1);
```

Comment:
Maybe this is even quicker?

```suggestion
	const [source] = generateSources(image, [
		{ breakpoint: breakpoints.desktop, width: 120 },
	]);
```